pkgs/by-name/su/supermariowar/package.nix

```
@@ -0,0 +1,68 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  pkg-config,
+  enet,
+  yaml-cpp,
+  SDL2,
+  SDL2_image,
+  SDL2_mixer,
+  zlib,
+}:
```

```suggestion
  unstableGitUpdater,
}:
```

You forgot to add it on top, which is why eval is failing.